test/unit/lib/plugins/aws/invoke.test.js

```diff
@@ -87,6 +178,8 @@ describe('AwsInvoke', () => {
       };
       awsInvoke.options.data = null;
       awsInvoke.options.path = false;
+      awsInvoke.options.context = null;
```

Comment:
Let's not edit and extend those tests (they're scheduled for full rewrite, and  new`runServerless` based tests should cover all we need)